spec/hashTable.js

```diff
@@ -0,0 +1,171 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable'
+
+chai.use( chaiChange )
+
+describe( 'HashTable', () => {
+
+	it( 'exists', () => {
+		expect( HashTable ).to.be.a( 'function' )
+	})
+
+	let myHashTable, testArray
+
+	beforeEach( () => {
+		myHashTable = new HashTable()
+	})
+
+	context( '#put( key, value )', () => {
```

Comment:
`context` should be an english statement about the state of the world

`context('when it has the values {a:1,b:2}', function(){ …`